docs/source/extensions/logging.rst

```diff
@@ -15,8 +15,24 @@ Logging
 #######
 
 Lightning supports the most popular logging frameworks (TensorBoard, Comet, etc...).
-To use a logger, simply pass it into the :class:`~pytorch_lightning.trainer.trainer.Trainer`.
-Lightning uses TensorBoard by default.
+
+By default Lightning uses `TensorBoard <https://www.tensorflow.org/tensorboard>`__ logging, storing logs to a directory (by default in ``lightning_logs/``).
```

Comment:
```suggestion
By default, Lightning uses `PyTorch TensorBoard <https://pytorch.org/docs/stable/tensorboard.html>`__ logging which uses `Tensorflow TensorBoard <https://www.tensorflow.org/tensorboard>`__ under the hood, and stores the logs to a directory (by default in ``lightning_logs/``).
```
how about linking the PyTorch tensorbord page which contains API documentation as well??

Reply:
@SeanNaren

Reply:
I think it would be good enough to just link the PyTorch Tensorboard page. As they have talked and linked Tensorboard on their page. @SeanNaren